src/Common/src/Interop/Windows/NtDll/Interop.RtlGetVersion.cs

```diff
@@ -11,12 +11,17 @@ internal partial class NtDll
         [DllImport(Libraries.NtDll, ExactSpelling=true)]
         private static extern int RtlGetVersion(ref RTL_OSVERSIONINFOEX lpVersionInformation);
 
-        internal static unsafe string RtlGetVersion()
+        internal static unsafe int RtlGetVersionEx(out RTL_OSVERSIONINFOEX osvi)
         {
-            var osvi = new RTL_OSVERSIONINFOEX();
+            osvi = new RTL_OSVERSIONINFOEX();
             osvi.dwOSVersionInfoSize = (uint)sizeof(RTL_OSVERSIONINFOEX);
+            return RtlGetVersion(ref osvi);
+        }
+
+        internal static unsafe string RtlGetVersion()
+        {            
             const string version = "Microsoft Windows";
```

Comment:
Nit: Again, I realize it was already like this, but since you're editing this anyway, we generally make consts be uppercase, e.g. `version` => `Version`